src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Ball {
+    private String ballColor;
+    private int ballNumber;
```

Comment:
```suggestion
    private String color;
    private int number;
```
these fields belong to the Ball class, so adding `ball` to the name of these fields is redundant